addons/stock/models/stock_move.py

```diff
@@ -1624,11 +1562,11 @@ def _prepare_procurement_values(self):
             'date_deadline': self.date_deadline,
             'move_dest_ids': move_dest_ids,
             'group_id': group_id,
-            'route_ids': self.route_ids,
+            'route_ids': self.route_ids or self.move_line_ids.result_package_id.package_type_id.route_ids,
             'warehouse_id': warehouse,
             'priority': self.priority,
             'orderpoint_id': self.orderpoint_id,
-            'product_packaging_id': self.product_packaging_id,
+            'uom_id': self.packaging_uom_id,
```

Comment:
Not sure it's required if no route can be set on the uom like it was on the packaging. And if it's required nonetheless, then call it `packaging_uom_id` to avoid confusion on what it is.